src/index.js

```diff
@@ -0,0 +1,60 @@
+import './style.css';
+
+const taskList = document.getElementById('taskList');
+
+const tasks = [
+  {
+    index: 1,
+    name: 'Reading book',
+    completed: false,
+  },
+  {
+    index: 2,
+    name: 'Meditating',
+    completed: false,
+  },
+  {
+    index: 3,
+    name: 'Riding bycicle',
+    completed: false,
+  },
```

Comment:
- [ ] Great job you did here. However, kindly use the specified  context used. Change the `name` to `Description` and fill appropriately  :

## Instructions:
![image](https://user-images.githubusercontent.com/50990925/220920964-919b40b8-5e28-4295-a3d6-a56418a277f9.png)